community/modules/database/slurm-cloudsql-federation/main.tf

```diff
@@ -49,6 +49,16 @@ resource "google_sql_database_instance" "instance" {
       ipv4_enabled                                  = false
       private_network                               = var.network_id
       enable_private_path_for_google_cloud_services = true
+
+      dynamic "authorized_networks" {
+        for_each = var.authorized_networks
+        iterator = ip_range
+
+        content {
+          name  = "ip-range-${ip_range.key}"
```

Comment:
AFAIK. `name` is optional and having `ip-range-<some arbitrary small int>` as an identifier have a questionable usability. 
Do you mind removing it or provide more information why it's desired ?

Reply:
Got it, this is fixed.